docs/tutorials/services/constrain-motion.md

```diff
@@ -0,0 +1,408 @@
+This configured table won't be taken into account by the Motion Service, but it's useful to be able to see it.
+
+On your robot's **Config** tab, create a new component called `table` with **Type** `generic` and **Model** `fake`.
+Click **Create component**, then click **Add frame**.
+
+Go to the **Frame System** subtab.
+Click **table** on the left menu.
+For **Geometry** click **box**.
+
+Set the dimensions (**Size**) to `1000` x `1000` x `20`.
+Notice that the top of the table is now 10mm above the origin, because the 20mm thickness is centered on the centerpoint.
+Account for this by giving the table a **Translation** of `-10` in the Z direction, moving the top of the table to 0.
+
+![The Frame System subtab of the Components tab with a 1000 wide, 1000 deep, 200 millimeter thick box representing the top of the table.](../../img/constrain-motion/frame-table.jpg)
+
+### Visualize obstacles on the table
+-->
+
+## Add an obstacle on the table
+
+In the last tutorial, you [added a table obstacle](../plan-motion-with-arm-gripper/#describe-the-robots-working-environment) to prevent the arm and gripper from hitting the table upon which the arm is mounted.
+In this tutorial, keep that table code and add an additional obstacle: a tissue box sitting on the table.
+
+We also added a `z_offset` parameter to the code so you can more easily calibrate your motion plans based on how high or low your arm is mounted compared to the table.
+
+You can adjust the dimensions and positions of the obstacles to describe your own scenario, but here is an example:
+
+```python {class="line-numbers linkable-line-numbers"}
+# Use this offset to set your z to calibrate based on where your table actually is
+z_offset = 10
+
+# Create a table obstacle
+table_origin = Pose(x=0.0, y=0.0, z=-19.0+z_offset)
+table_dims = Vector3(x=2000.0, y=2000.0, z=38.0)
+table_object = Geometry(center=table_origin, box=RectangularPrism(dims_mm=table_dims))
+
+# Create a tissue box obstacle
+box_origin = Pose(x=400, y=0, z=50+z_offset)
+box_dims = Vector3(x=120.0, y=80.0, z=100.0)
+box_object = Geometry(center=box_origin, box=RectangularPrism(dims_mm=box_dims))
+
+obstacles_in_frame = GeometriesInFrame(reference_frame="world",
+    geometries=[table_object, box_object])
+```
+
+## Determine the gripper's axes
+
+You need to figure out how the gripper's frame corresponds to its hardware.
+That is, you need to know which axis points from the "wrist" to the end of the gripper, which axis lines up with the direction the jaws actuate, and if the gripper was holding a cup, which axis would pass vertically through the cup.
+
+All example code below assumes +Z points from the base of the gripper to the point where its jaws close, +Y points towards the ground when the gripper is holding a cup from the side, and the X axis is the axis along which the jaws close, following the [right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule/).
```

Comment:
Two things:
1. The fake vs real is about grippers rather than arms. They could use a real arm and a fake gripper (which is almost what I did, but with a nonfunctioning gripper stand-in, configured as a fake). Not sure if that changes your feedback or if it was a typo.
2. In either case, this is actually relevant to all users. Even if they're using a fake arm AND a fake gripper, a big piece of understanding and using the frame system and motion service is thinking about how axes align with objects. Even though their fake gripper is just an arbitrary rectangular box, it's important to think about how it would work if it was a real gripper. If their gripper is fake, they can assign those axes however they want, but I'd hope they'd at least be imagining things like "okay, this represents this, so I need to move it this way" as a learning exercise.

Given this, do you think it'd be good if I changed the fake paragraph to "...there is no real hardware to calibrate and you can [continue this tutorial], imagining that your fake gripper corresponds to the diagram above."?

Reply:
1. Yes `("ohhh my arm actually grips along the Z axis", etc)` should have been `("ohhh my gripper actually grips along the Z axis", etc)`
2. I agree that that level of thinking is good here! I wasn't advising any deletions, just rearrangement, but you've convinced me that this is relevant to all users, and should remain where it is. I withdraw my suggestion! 👍

Reply:
Okay! Still, now that you've called more attention to this I think I want to add in the "imagining that your fake gripper corresponds to the diagram above." to tie it together a bit!